src/patternfly/components/Check/check-label.hbs

```diff
@@ -3,4 +3,7 @@
     {{{check-label--attribute}}}
   {{/if}}>
   {{> @partial-block}}
+  {{#if required}}
```

Comment:
We generally try to namespace all of the hbs params to the component and/or hbs file where they're used. In this case, since it's the check component is either required or not, that param could go on the main `{{#> check}}` block, like `{{#> check check--IsRequired}}`.

Then you can use `{{#if check--IsRequired}} {{> check-label-required}}{{/if}}` here and `{{#if check--IsRequired}} required{{/if}}` as an attribute in the check input hbs, similar to how we add the `checked` attribute: https://github.com/patternfly/patternfly/blob/9ab78a81b9d6d63cd7795fa82b1ff673abba5332/src/patternfly/components/Check/check-input.hbs#L3-L5

Reply:
Can we discuss it tomorrow? I just want to make sure I understand correctly. Thanks.